### PR TITLE
Refacto aws route53 resources

### PR DESCRIPTION
**Pull request by eliecharra**

### CODE REVIEW

File: pkg/resource/aws/aws_route53_health_check.go [hunk 1]
```diff
@@ -41,3 +44,9 @@ func (r *AwsRoute53HealthCheck) TerraformType() string {
 func (r *AwsRoute53HealthCheck) CtyValue() *cty.Value {
 	return r.CtyVal
 }
+
+func initAwsRoute53HealthCheckMetaData(resourceSchemaRepository resource.SchemaRepositoryInterface) {
+	resourceSchemaRepository.SetNormalizeFunc(AwsRoute53HealthCheckResourceType, func(val *resource.Attributes) {
+
+	})
```
Comment (wbeuil) [inline pkg/resource/aws/aws_route53_health_check.go:51]: Why is this empty ? And if it is intentional, why ?

File: pkg/middlewares/route53_records_id_reconcilier.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+// From Terraform state, we retrieve: 1234_toto_TXT
+type Route53RecordIDReconcilier struct{}
+
+func NewRoute53RecordIDReconcilier() Route53RecordIDReconcilier {
+	return Route53RecordIDReconcilier{}
+}
+
+func (m Route53RecordIDReconcilier) Execute(_, resourcesFromState *[]resource.Resource) error {
+
+	for _, remoteResource := range *resourcesFromState {
```
Comment (wbeuil) [inline pkg/middlewares/route53_records_id_reconcilier.go:24]: Should be `stateResource` instead of `remoteResource`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: pkg/middlewares/route53_records_id_reconcilier.go (hunk 2 above)
Comment (eliecharra) [reply pkg/middlewares/route53_records_id_reconcilier.go:24]: Nice catch thanks

File: pkg/resource/aws/aws_route53_health_check.go (hunk 1 above)
Comment (eliecharra) [reply pkg/resource/aws/aws_route53_health_check.go:51]: I need to remove it nice catch thanks

**Pull request merged**